### PR TITLE
Replace legacy build pipeline with PowerShell SDP build pipeline

### PR DESCRIPTION
**Pull request by qinezh**
Rebased version for PR https://github.com/MicrosoftDocs/PowerShell-Docs/pull/6227

### CODE REVIEW

File: reference/docfx.json [hunk 1]
```diff
@@ -162,6 +162,7 @@
                 "**/PSWorkflow/**.yml": "SteveL-MSFT",
                 "**/PSWorkflow/**.md": "SteveL-MSFT",
                 "**/PSWorkflowUtility/**.yml": "SteveL-MSFT",
+                "**/ThreadJob/**.md": "PaulHigin",
```
Comment (qinezh) [inline reference/docfx.json:165]: Need to replace the `.md` with `.yml`, thus 
```
"**/ThreadJob/**.yml": "PaulHigin"
```

File: reference/docfx.json [hunk 2]
```diff
@@ -199,6 +200,7 @@
                 "**/PSWorkflow/**.yml": "slee",
                 "**/PSWorkflow/**.md": "slee",
                 "**/PSWorkflowUtility/**.yml": "slee",
+                "**/ThreadJob/**.md": "paulhi",
```
Comment (qinezh) [inline reference/docfx.json:203]: same here

File: reference/docfx.json (hunk 1 above)
Comment (sdwheeler) [reply reference/docfx.json]: fixed

File: reference/docfx.json (hunk 2 above)
Comment (sdwheeler) [reply reference/docfx.json]: fixed

**Pull request merged**